### PR TITLE
ne: fix build on Linux

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by SMillerDev** [comment]
needs a `uses_from_macos "ncurses"`

**Comment by cho-m** [comment]
Might need a PTY:
```
==> /home/linuxbrew/.linuxbrew/Cellar/ne/3.3.1/bin/ne --macro /tmp/ne-test-20220426-10229-1ldfqx7/macros /tmp/ne-test-20220426-10229-1ldfqx7/test.txt
Cannot reopen input tty
Error: ne: failed
```

File: Formula/ne.rb [hunk 1]
```diff
@@ -38,7 +53,7 @@ def install
       InsertLine
       Exit
     EOS
-    system "script", "-q", "/dev/null", bin/"ne", "--macro", macros, document
+    PTY.spawn(bin/"ne", "--macro", macros, document)
```
Comment (cho-m) [inline Formula/ne.rb:56]: May want to try to wait for process to finish with something like `Process.wait pid`

Comment (danielnachun) [reply Formula/ne.rb]: That was the issue - I was actually able to use `sleep 1` and that was enough for it to work.  This should be ready to merge now.

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**